COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target
 ExecStart=/bin/sh -c "/bin/evmserver.sh start"
 ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
 Type=forking
+Restart=on-failure
```

Comment:
Ok, maybe that's enough for now then?  We can always refine this if it becomes a problem.